<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KCamView</class>
 <widget class="QWidget" name="KCamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string>                                        No Available Image View Yet</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QComboBox" name="camComboBox"/>
       </item>
       <item>
        <widget class="QListWidget" name="camList">
         <item>
          <property name="text">
           <string>Display Raw Image View</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Display Segmented Image View</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
